tests/parity/test_v4_preflights.py

```
@@ -0,0 +1,139 @@
+    (host / "scripts").mkdir()
+    (host / "scripts" / "run_tests.sh").write_text("#!/bin/sh\nexit 0\n", encoding="utf-8")
+    (host / "scripts" / "run_tests.sh").chmod(0o755)
+    for spec in v4_preflights.PREFLIGHT_NODE_MAP.values():
+        for node in spec.nodes:
+            root = plugin if node.owner == "plugin" else host
+            source = root / node.path
+            source.parent.mkdir(parents=True, exist_ok=True)
+            if not source.exists():
+                source.write_text("def test_fixture():\n    assert True\n", encoding="utf-8")
+    return plugin, host
+def _fake_run(monkeypatch: pytest.MonkeyPatch, *, plugin: Path, host: Path, output: str = "1 passed in 0.01s\n", returncode: int = 0, seen: list[tuple[tuple[str, ...], dict[str, str]]] | None = None, dirty: bool = False) -> None:
```

Ruff (0.16.3)

[warning] Unused function argument: `host`

(ARG001)